### PR TITLE
Add the OSLC Prolog README

### PR DESCRIPTION
**Pull request by berezovskyi**

### CODE REVIEW

File: oslc_prolog/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# OSLC prolog library
+
+This is a CPACK library for Cliopatria, follows the standard structure. It has support for programmatic OSLC resource manipulation, and custom OSLC service definition (via declaring `oslc_*` handlers with priorities), basic REST resource operations (POST to creation factory, PUT with matching `If-Match` header, GET with `oslc.properties` and `oslc.prefix` parameters, and DELETE). Resource shape validation supports occurs and type checks. REST interface supports all RDF+XML and TURTLE content types on both input and output.
+
+
+## Structure
+
+The library consists of the following directories:
+
+* `/snippets` is an extra folder with the examples.
```
Comment (likelion) [inline oslc_prolog/README.md]: /snippets is not even committed, please, remove

File: oslc_prolog/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+* `/rdf` contains the metadata.
+* `/rdf/cpack` contains the CPACK metadata.
+* `/rdf/base` contains data that will be preloaded with the cpack. Also contains a standard manifest ttl that defines how the preloading shall be done. In practice, this will load the `oslc` and `oslc_shape` graphs when this cpack is loaded.
+* `/config/` is needed for running in Clio.
```
Comment (likelion) [inline oslc_prolog/README.md]: there is no such folder, please, remove

File: oslc_prolog/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+* `/config-available` declares libraries to be loaded at the startup. We load both the libraries here as well as the application server.
+* `/lib/` contains the library code itself.
+* `/application` contains a sample server that listens on a predefined path.
```
Comment (likelion) [inline oslc_prolog/README.md]: it contains OSLC core server and dispatching mechanism to custom handlers of REST requests towards OSLC resources (there is no sample server, it is _the_ OSLC server).

**Comment by berezovskyi** [comment]
Please review again @likelion

**Review by likelion — APPROVED** [review]

**Pull request merged**